apps/passport-server/test/generic-issuance/pipelines/auto-issuance/setupAutoIssuancePipeline.ts

```diff
@@ -0,0 +1,286 @@
+      (product) => product.name === "auto-issuance-attendee-product"
+    );
+  expectToExist(autoIssuanceAttendeeProduct);
+  const autoIssuanceBouncerProduct =
+    autoIssuanceConfiguredEvents[0].products.find(
+      (product) => product.name === "auto-issuance-bouncer-product"
+    );
+  expectToExist(autoIssuanceBouncerProduct);
+  const autoIssuanceFoodVoucherProduct =
+    autoIssuanceConfiguredEvents[0].products.find(
+      (product) => product.name === "auto-issuance-food-voucher-product"
+    );
+  expectToExist(autoIssuanceFoodVoucherProduct);
+  const autoIssuanceFoodVendorProduct =
+    autoIssuanceConfiguredEvents[0].products.find(
+      (product) => product.name === "auto-issuance-food-vendor-product"
+    );
+  expectToExist(autoIssuanceFoodVendorProduct);
+
+  const autoIssuanceSemaphoreGroupIds = {
+    all: randomUUID(),
+    bouncers: randomUUID(),
+    attendees: randomUUID(),
+    attendeesAndBouncers: randomUUID()
+  };
+
+  const autoIssuancePipeline: PretixPipelineDefinition = {
+    ownerUserId: autoIssuanceGIUserID,
+    timeCreated: new Date().toISOString(),
+    timeUpdated: new Date().toISOString(),
+    id: randomUUID(),
+    editorUserIds: [],
+    options: {
+      feedOptions: {
+        feedDescription: "Auto Issuance Feed",
+        feedDisplayName: "Auto Issuance",
+        feedFolder: "Auto Issuance",
+        feedId: "auto-issuance"
+      },
+      events: autoIssuanceConfiguredEvents,
+      userPermissions: [
+        {
+          members: [
+            {
+              eventId: autoIssuanceEventId,
+              productId: autoIssuanceFoodVendorProduct.genericIssuanceId
+            }
+          ],
+          canCheckIn: {
+            eventId: autoIssuanceEventId,
+            productId: autoIssuanceFoodVoucherProduct.genericIssuanceId
+          }
+        }
+      ],
+      autoIssuance: [
+        {
+          memberCriteria: [
+            {
+              eventId: autoIssuanceEventId,
+              productId: autoIssuanceAttendeeProduct.genericIssuanceId
+            }
+          ],
+          eventId: autoIssuanceEventId,
+          productId: autoIssuanceFoodVoucherProduct.genericIssuanceId,
+          quantity: 2,
+          schedule: {
+            startDate: new Date().toISOString(),
+            intervalMs: 60_000,
+            endDate: new Date(Date.now() + 60_000 * 2 + 1).toISOString()
+          }
+        }
+      ],
+      manualTickets: [
+        {
+          id: randomUUID(),
+          eventId: autoIssuanceEventId,
+          productId: autoIssuanceAttendeeProduct.genericIssuanceId,
+          attendeeEmail: AutoIssuanceManualAttendeeEmail,
+          attendeeName: "Manual Attendee"
+        },
+        {
+          id: randomUUID(),
+          eventId: autoIssuanceEventId,
+          productId: autoIssuanceBouncerProduct.genericIssuanceId,
+          attendeeEmail: AutoIssuanceManualBouncerEmail,
+          attendeeName: "Manual Bouncer"
+        }
+      ],
```

Comment:
configuration that enables the auto-issuance and more refined permissions model for the auto issuance pipeline test